src/App.vue

```diff
@@ -227,15 +227,21 @@ export default {
     /** The route breadcrumbs list. */
     breadcrumbs (): Array<BreadcrumbIF> {
       const breadcrumbs = this.$route?.meta?.breadcrumb
-
-      return [
-        this.isRoleStaff ? staffDashboardBreadcrumb : dashboardBreadcrumb,
+      const crumbs = [
         {
           text: this.getEntityName || this.getCorpTypeNumberedDescription(this.getEntityType),
           to: { name: Routes.DASHBOARD }
         },
         ...(breadcrumbs || [])
       ]
+
+      if (this.isRoleStaff) {
```

Comment:
Please add a comment on why this code is needed.

Reply:
(what it's doing)

Reply:
Added in latest commit